Makefile

```diff
@@ -249,7 +249,7 @@ tag: check_branch
 
 .PHONY: integration_test
 integration_test: check_python check_branch $(project_root)/lambdas/service/.chalice/config.json
-	python -m unittest -v integration_test
+	python -m unittest --verbose --durations 0 integration_test
```

Comment:
Let's not log the duration for tests taking under 0.5s.

Reply:
For `make test` the equivalent `N slowest test cases` setting is roughly N=200.
For `make integration_test` I did not limit the output as there are 16 tests logged and only about 4-5 of these fall under 0.5s.

Reply:
Apologies, but I changed my mind: we should always log the durations of all tests.